hazelcast/test/src/sql_test.cpp

```diff
@@ -752,6 +783,10 @@ TEST_F(SqlTest, simple)
 
 TEST_F(SqlTest, rows_can_be_used_even_after_the_result_is_destroyed)
 {
+    // `VALUES` clause is not supported before 5.0.0
+    if (cluster_version() < member::version{ 5, 0, 0 })
```

Comment:
Why not put these into `SetUp` method? Can we eliminate repetition with this?

Reply:
Some specific tests should be disabled, `SetUp()` is not good place for it.